apps/pie-docs/src/_11ty/shortcodes/usage.js

```diff
@@ -0,0 +1,87 @@
+const pieIconsSvg = require('../filters/pieIconsSvg');
+const pieDesignTokenColours = require('../filters/pieDesignTokenColours');
+const list = require('./list');
+const usageTypes = require('../../_data/usageTypes');
+
+const metadata = {
+    do: {
+        displayName: 'Do',
+        styleColour: 'support-positive',
+        iconFill: 'support-positive',
+        iconName: 'check-circle',
+    },
+    dont: {
+        displayName: "Don't",
+        styleColour: 'support-error',
+        iconFill: 'support-error',
+        iconName: 'close-circle',
+    },
+};
+
+const buildImage = ({
+    width, alt, mobileSrc, src,
+}) => {
+    const isImageFullContainerWidth = !width;
+    const imageStyles = !isImageFullContainerWidth ? `style="--img-width: ${width};"` : ''; // If image isn't full width, set it to required width
+    const imageAlt = `alt="${alt || ''}"`;
+    const mobileImageMaxWidth = '600px';
```

Comment:
Could this be in the css files?

Reply:
I believe we can't as this value will need to be applied to the media attribute of the source tag and I'm not aware of a way to do that via CSS.